test/sequential/rolldown.withdraw.preOperation.test.ts

```diff
@@ -0,0 +1,578 @@
+    expect(eventManually.range.from.toNumber()).toEqual(nextRequestIdEth);
+    expect(eventManually.range.to.toNumber()).toEqual(
+      nextRequestIdEth + (batchSize - 2),
+    );
+    await Sudo.batchAsSudoFinalized(
+      ...(await Rolldown.createABatchWithWithdrawals(
+        testUser,
+        gaspIdL1Asset.ethereum,
+        1,
+      )),
+    );
+    const eventAutomatically = await Rolldown.waitForNextBatchCreated(
+      "Ethereum",
+      waitingBatchPeriod,
+    );
+    const sequencersList = await SequencerStaking.activeSequencers();
+    expect(sequencersList.toHuman().Ethereum).toContain(
+      eventAutomatically.assignee,
+    );
+    expect(eventAutomatically.source).toEqual("PeriodReached");
+    expect(eventAutomatically.range.from.toNumber()).toEqual(
+      nextRequestIdEth + (batchSize - 1),
+    );
+    expect(eventAutomatically.range.to.toNumber()).toEqual(
+      nextRequestIdEth + (batchSize - 1),
+    );
+  });
+
+  test("GIven a utility.batch ( batched tx of 35 and last item in the utility.batch is the mm_on ) When maintenance mode, THEN No automatic batch can happen", async () => {
+    await Sudo.batchAsSudoFinalized(
+      ...(await Rolldown.createABatchWithWithdrawals(
+        testUser,
+        gaspIdL1Asset.ethereum,
+        batchSize,
+      )),
+      Maintenance.switchMaintenanceModeOn(),
+    );
+    let error: any;
+    try {
+      await Rolldown.waitForNextBatchCreated("Ethereum", 10);
+    } catch (err) {
+      error = err;
+    }
+    expect(error).toEqual(
+      "method rolldown.TxBatchCreated not found within blocks limit",
+    );
+    await Sudo.batchAsSudoFinalized(Maintenance.switchMaintenanceModeOff());
+  });
+
+  test("Creating two merkle trees generates the same output", async () => {
+    const batchIdBefore = await getLastBatchId();
+    const withEvent = await Sudo.batchAsSudoFinalized(
+      ...(await Rolldown.createABatchWithWithdrawals(
+        testUser,
+        gaspIdL1Asset.ethereum,
+        10,
+      )),
+    );
+    const filteredEvent = JSON.parse(
+      JSON.stringify(
+        withEvent.filter(
+          (event) => event.method === "WithdrawalRequestCreated",
+        ),
+      ),
+    );
+    const idFromNumber = filteredEvent[0].event.data[1].id;
+    const idToNumber = filteredEvent[9].event.data[1].id;
+    const sequencer = JSON.parse(
+      JSON.stringify(await SequencerStaking.activeSequencers()),
+    ).Ethereum[0];
+    await Sudo.batchAsSudoFinalized(
+      Sudo.sudo(
+        await Rolldown.createForceManualBatch(
+          idFromNumber,
+          idToNumber,
+          sequencer,
+          "Ethereum",
+        ),
+      ),
+      Sudo.sudo(
+        await Rolldown.createForceManualBatch(
+          idFromNumber,
+          idToNumber,
+          sequencer,
+          "Ethereum",
+        ),
+      ),
+    );
+    const batchIdAfter = await getLastBatchId();
+    expect(batchIdAfter).toEqual(batchIdBefore + 2);
```

Comment:
plz check the merkle_tree data from the RPC calls, so that the root and proof hex are the same

Reply:
Done